src/levelset/levelSetObject.hpp

```diff
@@ -179,6 +179,7 @@ friend class LevelSetProxyObject;
     virtual void                                dump(std::ostream &);
     virtual void                                restore(std::istream &);
 
+    virtual LevelSetIntersectionStatus          _intersectInterfaceSurface(long id, int intrLocalId, double tolerance) const;
```

Comment:
All these new functions receive in input and interface. However, we are passing the id of a cell and a local id of the interface. Can we just pass the id of the interface?

Reply:
Done